### PR TITLE
Add visualization_user_name to fix #9134 cartodb issue.

### PR DESCRIPTION
**Pull request by donflopez**
Add visualization_user_name to fix #9134 cartodb issue, I added this to CartoDB/cartodb.js#1435.

cc @alonsogarciapablo

### CODE REVIEW

File: dist/torque.uncompressed.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
-Torque 2.15.0
-Temporal mapping for CartoDB
+Torque 2.15.2
+Temporal mapping for CARTO
```
Comment (javisantana) [inline dist/torque.uncompressed.js]: try to not mix general changes with bugfixes

File: dist/torque.uncompressed.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /**
```
Comment (javisantana) [inline dist/torque.uncompressed.js]: try to keep dist files outside of PR

File: dist/torque.uncompressed.js (hunk 1 above)
Comment (javisantana) [inline dist/torque.uncompressed.js]: looks like this is auotmatically generated :D

**Comment by pabletecodes** [comment]
Vamos @donflopez!

**Pull request closed without merging**